build_tools/build_wheels.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Build the package
+chmod u+x build_tools/build_package.sh
+./build_tools/build_package.sh
+
+# The version of the built dependencies are specified
+# in the pyproject.toml file, while the tests are run
+# against the most recent version of the dependencies
+pwd
```

Comment:
Are these for debugging the directory structure inside the container? If yes, could you please add that as an explanation?